### ISSUE TITLE
Missing instruction about CSS

### ISSUE THREAD

**Issue by Victoria91** [mdbootstrap/Tailwind-Elements#9]
Hello, thanks for the library!
I've followed your instructions from [here](https://tailwind-elements.com/quick-start/), but could not get the date picker to work. As I've inspected, it requires some specific CSS, so could you specify the correct way to include all necessary styles?

Thanks

**Comment by ErnestDaDev** [comment]
Same issue...using React. You need to detail how to use it with React because the imports are not working

**Comment by FelipeEndo** [comment]
Same here with React. many CSS classes missing, like `far`, `fa-calendar` and `datepicker-toggle-icon`.
I'm using Tailwind 3.0 and React 17

**Comment by emorevival** [comment]
Yeah I'm also having problems getting started with this on NextJS using TS, getting an error from `/dist/js/index.min.js` about `document` not being defined

**Comment by smolenski-mikolaj** [comment]
@Victoria91 @BesavedTech @FelipeEndo 

1. Have you installed the latest 1.0.0-alpha8 version?
2. Did you add the following configuration to your tailwind.config.js file?

```
module.exports = {
  content: ['./src/**/*.{html,js}', './node_modules/tw-elements/dist/js/**/*.js'],
  plugins: [
    require('tw-elements/dist/plugin')
  ]
}
```

Especially the content part with tw-elements js file is important, because some classes are added dynamically.

**Comment by emorevival** [comment]
On NextJS I have fixed the issue by putting the import in a `useEffect`

`  useEffect(() => {
    import('tw-elements');
  }, []);`

**Comment by hamzahejaz** [comment]
Can't use the time picker in react although included all of the files as described
